alf/algorithms/actor_critic_loss.py

```diff
@@ -81,13 +106,23 @@ def __init__(self,
         self._use_gae = use_gae
         self._lambda = td_lambda
         self._use_td_lambda_return = use_td_lambda_return
+        if normalize_scalar_advantages:
+            self._adv_norm = torch.nn.BatchNorm1d(
+                num_features=1,
+                eps=1e-8,
+                momentum=advantage_norm_momentum,
+                affine=False,
+                track_running_stats=True)
+            normalize_advantages = False
+        elif normalize_advantages:
+            self._adv_norm = torch.nn.BatchNorm1d(
+                num_features=reward_dim,
+                eps=1e-8,
+                momentum=advantage_norm_momentum,
+                affine=False,
+                track_running_stats=True)
```

Comment:
I found that changing this to True could lead to a mismatch between the layer and the ckpt which does not have keys ``running_mean`` and ``running_var``. I wonder if we might want to keep it False and config in conf files instead?

Reply:
You can try the following:
alf.config('Checkpointer.load', strict=False)

Reply:
I gave it a try in both the agent and expert conf files, it does not work out-of-the-box, will dig further to figure out why.